### PR TITLE
Add swoole-clear-error for swoole

### PR DESCRIPTION
**Pull request by sy-records**

### CODE REVIEW

File: reference/swoole/functions/swoole-clear-error.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+  </para>
```
Comment (cmb69) [inline reference/swoole/functions/swoole-clear-error.xml:30]:
```suggestion
  &return.void;
```

File: reference/swoole/functions/swoole-clear-error.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <type>void</type><methodname>swoole_clear_error</methodname>
+   <void />
+  </methodsynopsis>
+  <para>
+
```
Comment (cmb69) [inline reference/swoole/functions/swoole-clear-error.xml:17]: Please describe the function here; at the very least duplicate the refpurpose:
```suggestion
   Clear errors in the socket or on the last error code.
```

**Review by cmb69 — CHANGES_REQUESTED** [review]
Thank you for the PR! I left some comments below.

**Comment by sy-records** [comment]
Thanks~

**Comment by cmb69** [comment]
Thank you! I gave bad advice regarding `&return.void`; this needs to be enclosed in a `<para>`. I'm going to fix this, and apply the patch.

**Pull request closed without merging**